### PR TITLE
Phone Card field bug fix - CALS-5423

### PR DESCRIPTION
**Pull request by nasar-cambria**
#### Which User story does this relate to?
-https://osi-cwds.atlassian.net/browse/CALS-5423?filter=-1

#### Brief feature description
- Phone Card Not Working Properly When Re-Entering RFA-01A

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- N/A

### CODE REVIEW

File: app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx [hunk 1]
```diff
@@ -86,7 +87,7 @@ export default class ApplicantCard extends React.Component {
             idPrefix={idPrefix}
             phoneTypes={this.props.phoneTypes}
             applicant_id={this.props.index}
-            phones={this.props.applicantFields.phones || undefined}
+            phones={checkArrayObjectPresence(this.props.applicantFields.phones) || undefined}
```
Comment (ratneshraval) [inline app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx]: This bug should be corrected in API, when there is no phone number, it should return `null` not `[]`

Comment (Nasir2712) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx]: This bug would not have been logged if the api was sending a null value to begin with. So, now if they fix it, there is no need for us to wrap the props in that method.

Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx]: Agreed - That was my first approach but after discussing with Keith @lauherkcambria we decided to take this approach.

Comment (Nasir2712) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx]: also, does checkArrayObjectPresence handle null presence? if the api gets fixed in future?

Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCard.jsx]: I think it does. We can verify it inside ```commonHelper``` from where ```checkArrayObjectPresence``` is coming.

**Comment by ratneshraval** [comment]
Can you update PR title to include story number?

**Comment by nasar-cambria** [comment]
Updated story number.

**Comment by lauherkcambria** [comment]
@ratneshraval  @Nasir2712 we worked on this branch together with back end and came to agree together upon this solution. Lets get this merged please.

**Pull request merged**